openj9.test.sharedClasses.jvmti/src/test.sharedClasses.jvmti/net/openj9/stf/SharedClassesAPI.java

```diff
@@ -213,8 +213,15 @@ public void execute(StfCoreExtension test, StfSharedClassesExtension sharedClass
 				//    due to: https://github.com/eclipse-openj9/openj9-systemtest/issues/38 
 				if ( !PlatformFinder.isWindows() ) {
 					// Verify caches using a JVMTI native agent
-					String nativeExt    =  PlatformFinder.isWindows() ? ".dll" : ".so";
-					String nativePrefix =  PlatformFinder.isWindows() ? "" : "lib";
+					String nativeExt;
+					if (PlatformFinder.isOSX()) {
+						nativeExt = ".dylib";
+					} else if (PlatformFinder.isWindows()) {
```

Comment:
The whole block is inside `!PlatformFinder.isWindows()` so checking `PlatformFinder.isWindows() doesn't seem useful.